src/scss/global.scss

```diff
@@ -15,15 +15,16 @@ body,
 }
 
 body {
-  font: 1rem/1.3 "Rubik", "HelveticaNeue", "Helvetica Neue", Helvetica,
-    sans-serif;
+  font: 1rem/1.3 "speakTweak", "Rubik", "HelveticaNeue", "Helvetica Neue",
+    Helvetica, sans-serif;
+  // font-family: "speakTweak";
   -moz-osx-font-smoothing: grayscale;
   -webkit-font-smoothing: antialiased;
   background-color: $black;
   color: white;
   display: flex;
   justify-content: center;
-  font-family: "Rubik", sans-serif;
+  font-family: "speakTweak", sans-serif;
```

Comment:
This would be better set with a variable.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1240/commits/4b4351f2e7c0d9b970954dc3303bfa8db9db3a37